test/checkout.int.t.sol

```diff
@@ -13,17 +12,6 @@ abstract contract Checkout is UpgradeableMarket {
   uint256[] internal _removalIds;
   uint256 internal _certificateTokenId;
 
-  bytes32 constant RECEIVE_REMOVAL_BATCH_EVENT_SELECTOR =
-    keccak256(
-      "ReceiveRemovalBatch(address,address,uint256,uint256,uint256[],uint256[],address,uint256)"
-    );
-
-  function _deployMockERC20() internal returns (MockERC20Permit) {
-    MockERC20Permit impl = new MockERC20Permit();
-    bytes memory initializer = abi.encodeWithSignature("initialize()");
-    return MockERC20Permit(_deployProxy(address(impl), initializer));
-  }
-
```

Comment:
why is all of this code being deleted?  it looks like my tests that test purchases using an alternate ERC20 are also deleted from this file?